src/Identity/testassets/Identity.DefaultUI.WebSite/StartupBase.cs

```diff
@@ -104,18 +105,34 @@ public static void DisableFilePolling(IWebHostEnvironment env)
                     case IFileProvider staticWebAssets when staticWebAssets.GetType().Name == "StaticWebAssetsFileProvider":
                         GetUnderlyingProvider(staticWebAssets).UseActivePolling = false;
                         break;
+                    case IFileProvider manifestStaticWebAssets when manifestStaticWebAssets.GetType().Name == "ManifestStaticWebAssetFileProvider":
+                        foreach (var provider in GetUnderlyingProviders(manifestStaticWebAssets))
+                        {
+                            pendingProviders.Push(provider);
+                        }
+                        break;
                     case CompositeFileProvider composite:
                         foreach (var childFileProvider in composite.FileProviders)
                         {
                             pendingProviders.Push(childFileProvider);
                         }
                         break;
+                    case NullFileProvider:
+                        break;
                     default:
-                        throw new InvalidOperationException("Unknown provider");
+                        throw new InvalidOperationException($"Unknown provider '{currentProvider.GetType().Name}'");
                 }
             }
         }
 
+        private static IFileProvider[] GetUnderlyingProviders(IFileProvider manifestStaticWebAssets)
```

Comment:
Could we give the test IVT access instead? This feels very iffy

Reply:
I'll file a separate issue, I don't want to meddle with it on this PR.

Reply:
https://github.com/dotnet/aspnetcore/issues/34628